app/main.py

```diff
@@ -1,13 +1,33 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "name": [val for key, val in error.items() if key == "code"][0],
+        "line": [val for key, val in error.items() if key == "line_number"][0],
+        "column":
+            [val for key, val in error.items() if key == "column_number"][0],
+        "message": [val for key, val in error.items() if key == "text"][0],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors": [format_linter_error(i) for i in errors],
```

Comment:
i is not good.
We use 1 symbol names only if its a value used to track iteration
We use _ if value if not used
We name normally in every other cases